src/world/Spell/Customization/HackFixes.cpp

```diff
@@ -692,14 +692,6 @@ void ApplyNormalFixes()
 
         ARCEMU_ASSERT(sp->getSchool() < SCHOOL_COUNT);
 
-        // correct caster/target aura states
-        if (sp->getCasterAuraState() > 1)
-            sp->setCasterAuraState(1 << (sp->getCasterAuraState() - 1));
-
-        if (sp->getTargetAuraState() > 1)
-            sp->setTargetAuraState(1 << (sp->getTargetAuraState() - 1));
```

Comment:
Please explain this change.

Reply:
Well, Spell.dbc stores aurastate data in decimal since the aurastate fields can each have only one aurastate. The removed change here converted them to bitmask, which is not needed. A single spell cannot require more than one aurastate from caster or target.